package.json

```
@@ -92,7 +92,8 @@
 		"build:mac": "yarn run clean && electron-builder --mac dmg:x64 -p never",
 		"build:mac:arm64": "yarn run clean && electron-builder --mac dmg:arm64 -p never",
 		"build:win": "yarn run clean && electron-builder --win -p never",
-		"lint": "xo",
+		"check": "rome ci .",
+		"lint": "rome format . --write",
```

Formatting the whole source folder can create issues on Mac with the dist folder - how about making it run only on specific files?
```suggestion
		"lint": "rome format --write",
```
(then using `yarn lint FILE`)

---

I can just add the dist folder to rome.json .files.ignore

is that good enough?

---

I tried but it does not seem to work 😅 
```
Versions/Current/Resources internalError/fs ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Infinite symlink expansion
  
  ✖ Rome encountered a file system entry that leads to an infinite symbolic link expansion, causing an infinite cycle: Versions/Current/Resources
  

Versions/Current/Resources internalError/fs ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Infinite symlink expansion
  
  ✖ Rome encountered a file system entry that leads to an infinite symbolic link expansion, causing an infinite cycle: Versions/Current/Resources
  

A internalError/io ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ No such file or directory (os error 2)
  

Versions/Current/Resources internalError/fs ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Infinite symlink expansion
  
  ✖ Rome encountered a file system entry that leads to an infinite symbolic link expansion, causing an infinite cycle: Versions/Current/Resources
  

Versions/Current/Libraries internalError/io ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ No such file or directory (os error 2)
  

Versions/Current/Resources internalError/io ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ No such file or directory (os error 2)
  

Versions/Current/Helpers internalError/io ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ No such file or directory (os error 2)
  

A internalError/fs ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Infinite symlink expansion
  
  ✖ Rome encountered a file system entry that leads to an infinite symbolic link expansion, causing an infinite cycle: A
  

A internalError/fs ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Infinite symlink expansion
  
  ✖ Rome encountered a file system entry that leads to an infinite symbolic link expansion, causing an infinite cycle: A
  

A internalError/fs ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Infinite symlink expansion
  
  ✖ Rome encountered a file system entry that leads to an infinite symbolic link expansion, causing an infinite cycle: A
  

Formatted 0 file(s) in 2ms
Error: no files were processed in the specified paths.
```

I think it fails when listing the files, even if the whole folder is ignored

---

very weird... If that really is the case, then maybe you should submit a bug report to https://github.com/rome/tools

---

Wait so `yarn check` throws the same error for you?

---

Oh so actually it's a known bug with `rome format` and should be fixed in the next version

https://github.com/rome/tools/issues/4221

---

Same with `yarn check` but it does not prevent both commands from running correctly, so I guess we can live with it!